src/main/java/com/mide/gangsaeng/bannedword/BannedWordServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.mide.gangsaeng.bannedword;
+
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class BannedWordServiceImpl implements BannedWordService {
+
+    private static ConcurrentHashMap<String,BannedWord> cache;
+
+    public BannedWordServiceImpl() {
+        cache = new ConcurrentHashMap<>();
```

Comment:
Let's think when we do `new BannedWordServiceImpl` N times

Reply:
It is initialized every time the constructor is called. So i changed it to a **class variable**.